packages/terra-functional-testing/src/commands/viewport-helpers.js

```diff
@@ -0,0 +1,71 @@
+const Logger = require('../logger/logger');
+const terraViewports = require('../util/viewports');
+
+const logger = new Logger({ prefix: 'wdio-terra-service' });
+
+/**
+* Convenience method for getting the Terra test viewports by name.
+*
+* @param {string|string[]} sizes - the list of Terra viewport names.
+* @return {object} - Terra viewport sizes.
+*/
+const getViewports = (...sizes) => {
+  let viewportSizes = Object.keys(terraViewports);
+  if (sizes.length) {
+    viewportSizes = sizes;
+  }
+  return viewportSizes.map(formFactor => terraViewports[formFactor]);
+};
+
+/**
+* Sets the viewport for the test run to the Terra test viewport size.
+*
+* @param {string} [formFactor=huge] - the Terra test viewport.
+*/
+const setViewport = (formFactor = 'huge') => {
+  const terraViewport = terraViewports[formFactor];
+  if (terraViewport !== undefined && typeof terraViewport === 'object') {
+    global.browser.setWindowSize(terraViewport.width, terraViewport.height);
+  } else {
+    throw logger.error(`The ${formFactor} formFactor supplied is not a viewport size supported by Terra.`);
```

Comment:
This would `throw undefined`. The logger doesn't return an error. It logs a formatted error message to the terminal

Reply:
This should just log the message. https://github.com/cerner/terra-toolkit/pull/438/commits/19c190061a6d0f428d0ab4b33b669bed04d9f219